CONTRIBUTORS.md

```diff
@@ -732,7 +735,7 @@ GitHub contributors:
 - Sacha Froment
 - sadlyblue
 - sagaradonis
-- sallemiines
```

Comment:
Is this a rename or is it removed

Reply:
A rename, there is the long name on L351.

Reply:
Yep we have the full name now